frontend/src/styles/Team.css

```diff
@@ -0,0 +1,20 @@
+ .card {
+    height: auto;
+    margin : 3rem 1.5rem;
+    display: inline-flex;
+    text-align: center;
+    width: 13rem;
+    border: none;
+    vertical-align: auto;
+}
+.header {
+    text-align: center;
+    font-weight: 700;
+    font-size: 3em;
+}
+
+.container {
+    align-items: center;
+    display:block;
+}
+
```

Comment:
Consider giving the classnames a move unique name as leaving these as is will affect all components that will use container, header, and card classes.